pubspec.yaml

```diff
@@ -104,6 +114,15 @@ dev_dependencies:
 
 dependency_overrides:
   web: ^0.5.0
+  intl: 0.20.2
+  bitcoin_base:
+    git:
+      url: https://github.com/cake-tech/bitcoin_base
+      ref: cake-update-v16
+  blockchain_utils:
+    git:
+      url: https://github.com/cake-tech/blockchain_utils
+      ref: cake-update-v4
```

Comment:
uhm, no, I don't want us to update to this version of bitcoin base and blockchain utils yet, it's not stable, why is this needed? I don't see any new changes in that branch specific to enabling mweb in cupcake